docs/framework/whats-new/obsolete-members.md

```diff
@@ -224,14 +224,14 @@ The tables in this article list the type members that are obsolete in the .NET F
 |<xref:System.Security.Permissions.FileIOPermissionAttribute?displayProperty=nameWithType>|<xref:System.Security.Permissions.FileIOPermissionAttribute.All%2A>|Please use the <xref:System.Security.Permissions.FileIOPermissionAttribute.ViewAndModify%2A?displayProperty=nameWithType> property instead.|  
 |<xref:System.Security.Permissions.ReflectionPermissionAttribute?displayProperty=nameWithType>|<xref:System.Security.Permissions.ReflectionPermissionAttribute.ReflectionEmit%2A>|This permission is no longer used by the CLR.|  
 |<xref:System.Security.Permissions.ReflectionPermissionAttribute?displayProperty=nameWithType>|<xref:System.Security.Permissions.ReflectionPermissionAttribute.TypeInformation%2A>|This API has been deprecated.|  
-|<xref:System.Security.Permissions.ReflectionPermissionFlag?displayProperty=nameWithType>|<xref:System.Security.Permissions.ReflectionPermissionFlag>|This permission has been deprecated. The <xref:System.Security.Permissions.ReflectionPermissionFlag> enumeration member does not include <xref:System.Security.Permissions.ReflectionPermissionFlag>. Use <xref:System.Security.Permissions.PermissionState?displayProperty=nameWithType> to get full access.|  
-|<xref:System.Security.Permissions.ReflectionPermissionFlag?displayProperty=nameWithType>|<xref:System.Security.Permissions.ReflectionPermissionFlag>|This permission is no longer used by the CLR.|  
-|<xref:System.Security.Permissions.ReflectionPermissionFlag?displayProperty=nameWithType>|<xref:System.Security.Permissions.ReflectionPermissionFlag>|This API has been deprecated.|  
+|<xref:System.Security.Permissions.ReflectionPermissionFlag?displayProperty=nameWithType>|<xref:System.Security.Permissions.ReflectionPermissionFlag.AllFlags>|This permission has been deprecated. The <xref:System.Security.Permissions.ReflectionPermissionFlag> enumeration member does not include <xref:System.Security.Permissions.ReflectionPermissionFlag>. Use <xref:System.Security.Permissions.PermissionState?displayProperty=nameWithType> to get full access.|  
```

Comment:
third xref should be AllFlags too and 4th one should be System.Security.Permissions.ReflectionPermissionFlag.RestrictedMemberAccess and 5th should be System.Security.Permissions.PermissionState.Unrestricted